src/java.base/share/classes/java/text/NumberFormat.java

```diff
@@ -214,10 +225,14 @@
  * configured to round floating point numbers using half-even
  * rounding (see {@link java.math.RoundingMode#HALF_EVEN
  * RoundingMode.HALF_EVEN}) for formatting.
+ * </ul>
  *
+ * @spec         https://www.unicode.org/reports/tr35
+ *               Unicode Locale Data Markup Language (LDML)
  * @see          DecimalFormat
  * @see          ChoiceFormat
  * @see          CompactNumberFormat
+ * @see          Locale
```

Comment:
Could be removed as the link is now gone

Reply:
As this is a localized class, and the class description mentions `Locale` in various places, I feel like `Locale` should still be included as a `see` tag, (even though I removed the locale link). I am fine either way, what do you think?

Reply:
OK, let's keep them then